crates/sui-indexer-alt/src/pipeline/concurrent/mod.rs

```diff
@@ -56,8 +57,8 @@ pub trait Handler: Processor {
     const MIN_EAGER_ROWS: usize = 50;
 
     /// If there are more than this many rows pending, the committer will only commit this many in
-    /// one operation.
-    const MAX_CHUNK_ROWS: usize = 1000;
+    /// one operation. The size is chosen to maximize the rows without hitting the limit on bind parameters.
+    const MAX_CHUNK_ROWS: usize = i16::MAX as usize / Self::Value::FIELD_COUNT;
```

Comment:
Rather than leaving this as a constant which can be overridden, I would inline the computation in the concurrent collector implementation, because we don't really want people to be able to override this after your change.